psc-ide.el

```diff
@@ -500,8 +514,11 @@ passes it into the callback"
                    (start (cdr (assoc 'start position)))
                    (line (aref start 0))
                    (column (aref start 1)))
-              (require 'etags)
-              (ring-insert find-tag-marker-ring (point-marker))
+              (if (fboundp 'xref-push-marker-stack) ;; Emacs 25
+                  (xref-push-marker-stack)
```

Comment:
Don't want to "sneak" this in, but I forgot to merge this addition since it was only on my local computer. xref seems to be the new standard in emacs 25.